### DISCUSSION TITLE
Consider exposing `EpochInfo` via JSON RPC

### DISCUSSION THREAD

**Discussion by frol** [near/nearcore#5736] [Ideas] [answered]
@bowenwang1996 How do you feel about exposing [`EpochInfo`](https://github.com/near/nearcore/blob/9cc589308b4331a291c9a249c097c188e59cf1e9/core/primitives/src/epoch_manager.rs#L446-L518) to the external users? I got to this problem while discussing seat price, but I see that there are a few other potentially useful pieces of information (information about validators, rewards, minted tokens)

**Comment by nikurt** [comment]
+1 to exposing `EpochInfo` in the RPC interface.
I wanted to extend `state-viewer` to expose it as well.

**Comment by bowenwang1996** [comment, marked as answer]
Yeah sounds good to me
